desktop/src/Desktop/Frontend.hs

```diff
@@ -92,15 +99,62 @@ fileStorage dir = Storage
     where path :: Show a => a -> FilePath
           path k = dir </> FilePath.makeValid (show k)
 
-bipCrypto :: Crypto.XPrv -> Text -> Crypto Crypto.XPrv
+newtype XPactSecret = XPactSecret { unXPactSecret :: BS.ByteString }
```

Comment:
what is this, is it different from `PrivateKey`?

Reply:
It's going away, along with `DesktopKey`